config/routes.rb

```diff
@@ -12,6 +12,7 @@
   post '/order_cats', to: "order_cats#create"
   get '/order/:id', to: "orders#show", as: "order"
   delete '/cart_items', to: "cart_items#delete"
+  get '/orders', to: "orders#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.